Tests/XCUITests/HistoryTests.swift

```diff
@@ -6,8 +6,13 @@ import XCTest
 
 let webpage = ["url": "www.mozilla.org", "label": "Internet for people, not profit — Mozilla", "value": "mozilla.org"]
 let oldHistoryEntries: [String] = ["Internet for people, not profit — Mozilla", "Twitter", "Home - YouTube"]
+let emptyRecentlyClosedMesg = "Websites you’ve visited recently will show up here."
 // This is part of the info the user will see in recent closed tabs once the default visited website (https://www.mozilla.org/en-US/book/) is closed
-let closedWebPageLabel = "localhost:\(serverPort)/test-fixture/test-mozilla-book.html"
+let bookOfMozilla = ["file": "test-mozilla-book.html", "title": "The Book of Mozilla", "label": "localhost:\(serverPort)/test-fixture/test-mozilla-book.html"]
+
+// Default timeout for waitForExistence and waitForNoExistence
+let TIMEOUT: TimeInterval = 15
+let TIMEOUT_LONG: TimeInterval = 45
```

Comment:
These two lines should go in the BaseTestCase so that they can be used by all tests